### PR TITLE
update CODEOWNERS

### PR DESCRIPTION
**Pull request by ptrsmrn**
Removed people that no longer contribute to the scylladb.git and added/substituted reviewers responsible for maintaining the frontend components.

No need to backport, this is just an information for the github tool.

### CODE REVIEW

**Comment by ptrsmrn** [comment]
CC @kostja @nyh @nuivall @KrzaQ

**Comment by mykaul** [comment]
> Removed people that no longer contribute to the scylladb.git and added/substituted reviewers responsible for maintaining the frontend components.

Based on what did we determine they do not contribute anymore? A month of no contribution? A year? We need to have some guidelines here.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 # CACHE
 row_cache* @tgrabiec
 *mutation* @tgrabiec
 test/boost/mvcc* @tgrabiec
 
 # CDC
-cdc/* @kbr- @elcallio @piodul @jul-stas
-test/cql/cdc_* @kbr- @elcallio @piodul @jul-stas
-test/boost/cdc_* @kbr- @elcallio @piodul @jul-stas
+cdc/* @kbr- @elcallio @piodul
```
Comment (dawmd) [inline .github/CODEOWNERS:10]: `kbr-` -> `kbr-scylla`

Comment (ptrsmrn) [reply .github/CODEOWNERS:10]: @kbr-scylla ? @kbr- ?

**Comment by ptrsmrn** [comment]
> > Removed people that no longer contribute to the scylladb.git and added/substituted reviewers responsible for maintaining the frontend components.
> 
> Based on what did we determine they do not contribute anymore? A month of no contribution? A year? We need to have some guidelines here.

@mykaul 
Their last contribution was:
1. jul-stas - May '23
2. havaker - Oct '23
3. eliransin - Feb '24

As per our conversation, we decided to remove someone from the list if (s)he hasn't contributed/reviewed anything for the last year, which means I need to revert removing 'havaker' and 'eliransin'.

**Comment by ptrsmrn** [comment]
Per https://github.com/scylladb/scylladb/pull/20136#issuecomment-2288612029, restored `eliransin` (I should've restored `havaker` as well, but he's close to the deadline anyways, so...)

File: .github/CODEOWNERS (hunk 1 above)
Comment (ptrsmrn) [reply .github/CODEOWNERS]: Done as requested.

**Comment by ptrsmrn** [comment]
I also changed `kbr-` to `kbr-scylla`
cc @kbr-scylla

**Comment by ptrsmrn** [comment]
@avikivity can you consider merging this? If not, what needs to be changed?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -25,15 +25,15 @@ compaction/* @raphaelsc
...
+counters* @nuivall @ptrsmrn @KrzaQ
+tests/counter_test* @nuivall @ptrsmrn @KrzaQ
 
 # DOCS
 docs/* @annastuchlik @tzach
-docs/alternator @annastuchlik @tzach @nyh @havaker @nuivall
+docs/alternator @annastuchlik @tzach @nyh @nuivall @ptrsmrn @KrzaQ
 
 # GOSSIP
 gms/* @tgrabiec @asias
```
Comment (kbr-scylla) [inline .github/CODEOWNERS:39]: Should add me here.

Comment (ptrsmrn) [reply .github/CODEOWNERS]: Thanks, done.

**Comment by denesb** [comment]
> > > Removed people that no longer contribute to the scylladb.git and added/substituted reviewers responsible for maintaining the frontend components.
> > 
> > 
> > Based on what did we determine they do not contribute anymore? A month of no contribution? A year? We need to have some guidelines here.
> 
> @mykaul Their last contribution was:
> 
>     1. jul-stas - May '23
> 
>     2. havaker - Oct '23
> 
>     3. eliransin - Feb '24
> 
> 
> As per our conversation, we decided to remove someone from the list if (s)he hasn't contributed/reviewed anything for the last year, which means I need to revert removing 'havaker' and 'eliransin'.

None of these people work for us anymore, no point in keeping them in the code owners file.

**Comment by denesb** [comment]
> @scylladb/scylla-maint Can we push this PR or should I rather close it? Is there anything I should change here?

LGTM, queueing.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # AUTH
-auth/* @elcallio @vladzcloudius
+auth/* @nuivall @ptrsmrn @KrzaQ
```
[the remark this answers is not in the document]
Comment (ptrsmrn) [reply .github/CODEOWNERS:2]: > This mechanism only works with people who have write access

So if someone is not a maintainer, then practically nothing changes, huh. Thanks for letting me know. 
@dani-tweig just a thought: this flie (`.github/CODEOWNERS`) could be updated with SME spreadsheet. Maybe it'd be better to have core SMEs listed here.

[the remark this answers is not in the document]
Comment (dani-tweig) [reply .github/CODEOWNERS:2]: @ptrsmrn, I dont know if people, when looking for an expert, will turn to this file, but if you think they will, you are free to update it according to the SME spreadsheet.

**Pull request merged**